### PR TITLE
Add codesandbox template for react-next package

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This adds a codesandbox template for the react-next package

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/codesandbox-react-next-template/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+function start(): void {
+  if (!_rootDiv) {
+    _rootDiv = document.createElement('div');
+    document.body.appendChild(_rootDiv);
+  }
+
+  ReactDOM.render(
+    <Fabric>
```
[the remark this answers is not in the document]
Comment (xugao) [reply apps/codesandbox-react-next-template/src/index.tsx:39]: https://codesandbox.io/s/jovial-fermat-lmjxh

**Comment by xugao** [comment]
can we give meaning for sandbox names?
image

**Comment by tomi-msft** [comment]
unfortunately we are unable to make the names for the codesandbox templates unless we make them directly on the website, as the first one was done. David has pinged a codesandbox dev, and they are working to add that functionality, or at least make it so the codesandbox name is associated with the package name

**Comment by tomi-msft** [comment]
closing this to merge it with this PR: https://github.com/microsoft/fluentui/pull/13893

**Pull request closed without merging**